python/hail/utils.py

```diff
@@ -232,3 +232,13 @@ def wrap_to_list(s):
         return s
     else:
         return [s]
+
+def get_env_or_default(maybe, envvar, default):
+    import os
+
+    if (maybe != None):
+        return maybe
+    elif (envvar in os.environ and os.environ[envvar] != ""):
```

Comment:
can be `elif os.get(envvar):`

Reply:
I'm in python 2.7.12 and I don't have a `get` method on `os`, do I need to update? I needed the two conditions here because `os.environ[envvar]` on an `envvar` that is unset errors:
```python
>>> os.environ['TMPDIR']
'/var/folders/f_/58n921753xv9p69q7v3lgxb9s8n2n4/T/'
>>> os.environ['FOO']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/dking/anaconda2/lib/python2.7/UserDict.py", line 40, in __getitem__
    raise KeyError(key)
KeyError: 'FOO'
```

Reply:
Sorry, I made a goof! It's 
```python
elif os.environ.get(envvar):
```

dicts have a get method which returns `None` instead of raising a key error.